Static/DataSource.swift

```diff
@@ -188,11 +187,11 @@ extension DataSource: UITableViewDataSource {
     }
 
     public func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
     }
     
     public func tableView(tableView: UITableView, estimatedHeightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
```

Comment:
Sure that makes sense. That's part of the geometry calculation stuff I mentioned before. To be fair, it doesn't have to lay out _every_ cell, only the one that will initially be on screen. If it did, `tableView(_: cellForRowAtIndexPath:)` would be called for every row instead of on demand which would defeat the point of cell re-use.